src/sodiumCompatibility/java/net/coderbot/iris/compat/sodium/mixin/vertex_format/MixinSodiumWorldRenderer.java

```diff
@@ -21,6 +24,6 @@
                             ")Lme/jellysquid/mods/sodium/client/render/chunk/ChunkRenderBackend;"))
     private ChunkVertexType iris$overrideVertexType(ChunkVertexType vertexType) {
         // TODO: Don't use the extended vertex format when shaders are disabled.
```

Comment:
You can probably remove this comment since you're making the change that resolves it.

Reply:
Done